src/pkg/reg/adapter/tencentcr/adapter.go

```diff
@@ -40,13 +41,15 @@ import (
 
 var (
 	errInvalidTcrEndpoint = errors.New("[tencent-tcr.newAdapter] Invalid TCR instance endpoint")
+	rateLimiterTransport  http.RoundTripper
 )
 
 func init() {
-	var envTcrQPSLimit, _ = strconv.Atoi(os.Getenv("TCR_QPS_LIMIT"))
-	if envTcrQPSLimit > 1 && envTcrQPSLimit < tcrQPSLimit {
-		tcrQPSLimit = envTcrQPSLimit
+	var envTcrQPSLimit, _ = strconv.Atoi(os.Getenv("REG_ADAPTER_TCR_QPS_LIMIT"))
```

Comment:
@kofj Please take a look, because as this refactor, the env key of rate limit for TCR has been changed.